pandas/io/excel.py

```diff
@@ -226,7 +231,7 @@ def read_excel(io, sheet_name=0, header=0, skiprows=None, skip_footer=0,
 
     return io._parse_excel(
         sheetname=sheet_name, header=header, skiprows=skiprows, names=names,
-        index_col=index_col, parse_cols=parse_cols, parse_dates=parse_dates,
```

Comment:
I would change the internal kwargs all the way down to ``usecols``

Reply:
Ah, good point.  Will do.

Reply:
BTW, this PR is just addressing #4988, not closing it?

Reply:
how so?

Reply:
Well because the conversation arose from #4988, so at the very least we're addressing it.  However, I wasn't sure if we were closing the issue after this PR.

Reply:
I don't really remember this, can you give me a 1-liner what the issue is? (I thought it was just a rename / deprecation)

Reply:
The issue was regarding "inconsistent naming between `read_excel` and `read_csv`"  That was really it.  I didn't see anything naming inconsistencies besides this one.

Reply:
ok to close that issue (I changed at top)